### ISSUE TITLE
Pretrained rllib models?

### ISSUE THREAD

**Issue by leobxpan** [HumanCompatibleAI/human_aware_rl#10]
Thank you for the codebase. I'm trying to use a pretrained rllib agent to test the demo / environment but I didn't seem to find a proper one in the repo. Are there these model checkpoints available?

**Comment by micahcarroll** [comment]
I don't think we currently have pretrained rllib agents released in the repo – this is something that we were planning on doing soon anyways. @mesutyang97 or @nathan-miller23 might be able to do a quick run and release the hyperparams and model for one of the layouts. This is something that we were planning on doing soon anyways.

**Comment by mesutyang97** [comment]
Hi @leobxpan ,

Since the environment recently went through a lot of updates, we currently have not checkpoints for an agent for any of the layouts. I can do some quick runs later today, and release a checkpoint for `cramped_room` tomorrow (and possibly more in the future). 

We will also update the readme to include instructions to train and load your own checkpoints

**Comment by leobxpan** [comment]
Great to hear. Looking forward to them!

**Comment by nathan-miller23** [comment]
Looking at our repo structure, I think it makes more sense to store pre-trained agents in [this](https://github.com/HumanCompatibleAI/overcooked-demo) repo. Therefore I'm linking [this](https://github.com/HumanCompatibleAI/overcooked-demo/pull/24) PR, which currently includes some dummy agents, and will hopefully include a couple legit PPO agents once they finish training overnight
